File: Homework 1/turker_ege.dart
void main() {

  var playlists = { 'pop': 104,'rock': 243, 'jazz': 72, 'electronic': 24 };
  
  print(playlists['pop']);
  print(playlists['metal']);
  
  playlists['hiphop'] = 44;
  
  playlists.remove('rock');
  
  playlists['jazz'] = 83;
  
  print(playlists.containsKey('electronic'));
  print(playlists.containsKey('rock'));
  
  print(playlists.containsValue(104));
  print(playlists.containsValue(243));
  
  playlists.forEach((key,value) => foo(key,value)); 
  
}

void foo(key,value){
  print('${key}: ${value}');
}

File: Homework 1/turker_ege.rs
use std::collections::HashMap;
fn main() {
    let mut playlists = HashMap::new();
    
    playlists.insert("pop",104);
    playlists.insert("rock",243);
    playlists.insert("jazz",72);
    playlists.insert("electronic",24);
    
    println!("{}",playlists["pop"]);
    
    playlists.insert("hiphop",44);
    
    playlists.remove("rock");
    
    playlists.insert("jazz",83);
    
    println!("{}", playlists.contains_key("electronic"));
    println!("{}", playlists.contains_key("rock"));
    
    println!("{}", playlists.values()
        .any(|value| *value == 104));
    println!("{}", playlists.values()
        .any(|value| *value == 243));
    
    for(key, value) in playlists {
        foo(key,value);
    }
}

fn foo(key: &str,value: i32) {
    println!("{}: {}", key, value);
}